apollo-router/src/query_planner/caching_query_planner.rs

```diff
@@ -261,7 +261,7 @@ where
                 query: query.clone(),
                 operation: operation.clone(),
                 hash: doc.hash.clone(),
-                sdl: Arc::clone(&self.schema.raw_sdl),
+                schema_id: Arc::clone(&self.schema.hash),
```

Comment:
I renamed it to `schema_id` everywhere in [fa19613](https://github.com/apollographql/router/pull/5374/commits/fa19613a37f4b320a97b26784d638be1b7b04276) since we were already naming it that way elsewhere